nservicebus/azure-storage-queues/configuration.md

```diff
@@ -6,11 +6,7 @@ tags:
 - Cloud
 ---
 
-The default settings can be overridden by adding a configuration section called `AzureServiceBusQueueConfig` to the web.config or app.config files:
-
-snippet:AzureStorageQueueConfig
```

Comment:
No. The snippet is still valid up to version 6. For Version 7 we use code only configuration. I wanted to kill two birds with one stone and provide a generic description first, later, differentiate between the configuration section and the code only configuration. Does it answer your question?